### DISCUSSION TITLE
Error when using next/image in MDX files with next & @next/mdx version 13.5.4

### DISCUSSION THREAD

**Discussion by DavidBuck** [vercel/next.js#56514] [Help]
### Summary

Hi, I have been trying to get next/image working with MDX files. I used create-next-app@latest to set up a test project and followed the MDX configuration docs to render the MDX file. In the MDX file I imported a test image & added the next/image component. 

 If I run the project with the current next (13.5.4) & @next/mdx (13.5.4) versions I get the following error - 
 _Error: Cannot access Image.propTypes on the server. You cannot dot into a client module from a server component. You can only pass the imported name through. I get the same error with other 13.5.* versions._

However, If I run the project with "next": "13.4.19" and "@next/mdx": "13.4.19" (or other 13.4.* versions) the image is displayed correctly.

Is there something I need to change when using next/image with next & @next/mdx versions above 13.5?

Thanks for your help

### Additional information

```js
File path:
/app/my-mdx-page/image.jpg
/app/my-mdx-page/page.mdx

MDX file content:

import testImage from "./image.jpg";
import Image from "next/image";

# Welcome to my MDX page!


<Image src={testImage} />


package.json
```json
  "dependencies": {
    "@mdx-js/loader": "^2.3.0",
    "@mdx-js/react": "^2.3.0",
    "@next/mdx": "13.5.4",
    "next": "13.5.4",
    "react": "^18",
    "react-dom": "^18"
  },
```

mdx-components.js
```js
export function useMDXComponents(components) {
  return {
    ...components,
  };
}
```

next.config.js
```js
/** @type {import('next').NextConfig} */
const withMDX = require("@next/mdx")();

const nextConfig = {
  pageExtensions: ["js", "jsx", "mdx", "ts", "tsx"],
};

module.exports = withMDX(nextConfig);
```
```


### Example

_No response_

**Comment by Starefossen** [comment]
Stil broken i `v14.0.2`

**Reply by DavidBuck** [reply]
There is now an issue related to this https://github.com/vercel/next.js/issues/58888

I've found the error only occurs when running in development mode (`$ npx next dev`) but not in production mode (`$ npx next build && npx next start`).

**Comment by stevecastaneda** [comment]
Can confirm this is also happening to me in dev mode.  Reverting to 13.4.x fixes the issue.
